src/pages/fieldManagement/settings/saveField/OptionEditor.jsx

```diff
@@ -42,7 +41,7 @@ export default function OptionEditor({
                 flexDirection: 'column',
                 marginBottom: 12,
               }}
-              key={option.id}
+              key={option.value}
```

Comment:
Option values and/or labels cannot be guaranteed to be unique within this component. If a user clicks "Add Another Option" several times, each option looks exactly the same: `[{ label: '', value: '' }, { label: '', value: '' }, // etc.]`

Reply:
Also, "value" is not a stable identifier for a key because whenever an option's value is updated, the key changes. This causes the value input to lose focus.

Reply:
Hm...what would you recommend in this case? Just generate a uuid?

Reply:
Option values and/or labels cannot be guaranteed to be unique within this component. If a user clicks "Add Another Option" several times, each option looks exactly the same: [{ label: '', value: '' }, { label: '', value: '' }, // etc.]

I think that this will at least be partially ameliorated by Jon's back end work on DEX-1270 (this makes me realize that I mis-typed and this whole FE PR is for ticket DEX-1120).

Reply:
It will be partially ameliorated because we will be able to know that the values from the database are unique. However, while values are being added and edited within the dialog, it will always be possible for multiple value fields to be the same.

Reply:
once the values from the database are unique, it might be sufficient to add a timestamp property to objects created on the frontend ie. `{ label, value, created: Date.now() }`. obviously the backend objects wouldn't have these but you could use `key={o?.created || o?.value}`

Reply:
> Also, "value" is not a stable identifier for a key because whenever an option's value is updated, the key changes. This causes the value input to lose focus. 
for this to be resolved we would need to make sure the value used in the keys was the backend value, not the frontend's input value that changes as typed

Reply:
Ok... with the most recent changes, I believe that the only way that two options would have the same key would be if 1) they were migrated and 2) they had the same exact value. 

I would argue that the behavior of deletion of any one of these resulting in the deletion of all of them is a feature rather than a bug, because it helps improve the usability of the product.